packages/cli/src/handlers/compile.ts

```diff
@@ -15,14 +15,19 @@ import {
     warehouseCredentialsFromDbtTarget,
 } from '../dbt/profile';
 import * as styles from '../styles';
+import { dbtCompile, DbtCompileOptions } from './dbt/compile';
 
-type GenerateHandlerOptions = {
+type GenerateHandlerOptions = DbtCompileOptions & {
     projectDir: string;
     profilesDir: string;
     target: string | undefined;
     profile: string | undefined;
 };
-export const compile = async (options: GenerateHandlerOptions) => {
+export const compile = async (
+    options: GenerateHandlerOptions,
+    command: any,
+) => {
+    await dbtCompile(options, command.parent.args.slice(1));
```

Comment:
why do we need this `command` ? Aren't all the options for `dbt compile` in the `options` object?